packages/author-profile/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "@thetimes/with-client": "1.0.9",
-    "@times-components/author-profile": "0.58.5",
+    "@times-components/author-profile": "0.58.6",
```

Comment:
Why the author profile bump?

Reply:
That's not related to the pr but had to do it since I symlink to times-components, need to keep versions in sync